<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/icons/texture.png);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGroupBox" name="recordGroupBox">
        <property name="title">
         <string>Record</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Width :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>y :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="x">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:none;</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>x :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="width">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:none;</string>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="y">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:none;</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Height :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLineEdit" name="height">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">border:none;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="selectRegion">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string>Select Region</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/shape-2-line.png</normaloff>:/icons/shape-2-line.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="startRec">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true"> background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(41, 77, 128, 255), stop:1 rgba(65, 145, 245, 255));

border:none;
color: white;</string>
           </property>
           <property name="text">
            <string>Start Recording</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/white/white_record-circle-line.png</normaloff>:/icons/white/white_record-circle-line.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stopRec">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Ubuntu</family>
             <weight>75</weight>
             <bold>true</bold>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border:none; background-color: qlineargradient(spread:reflect, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(201, 0, 0, 255), stop:1 rgba(242, 0, 0, 255));
border:none;
color: white;</string>
           </property>
           <property name="text">
            <string>(00:00:00) Stop Recording</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/icons/white/stop-line.png</normaloff>:/icons/white/stop-line.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="styleSheet">
         <string notr="true">/*QGroupBox::title
{
    subcontrol-origin: margin;
    subcontrol-position: top center;
}
QGroupBox::indicator
{
    margin-left: 0.2ex;
}*/</string>
        </property>
        <property name="title">
         <string>Application settings</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="1" alignment="Qt::AlignLeft">
            <widget class="QCheckBox" name="darkTheme">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Dark theme :</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="Line" name="line_4">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>1</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <italic>true</italic>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>Selection settings</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>1</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="1" alignment="Qt::AlignLeft">
            <widget class="QCheckBox" name="decoration">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Decoration hint:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Selection border:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignLeft">
            <widget class="QSpinBox" name="selectionRectBorder">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="Line" name="line_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>1</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <italic>true</italic>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>Capture settings</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="Line" name="line_2">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>1</height>
              </size>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="3" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Capture rectangle visible :</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="selectionRectVisible">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Capture cursor :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignLeft">
            <widget class="QCheckBox" name="followCursor">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="captureCursor">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0" alignment="Qt::AlignRight">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Follow cursor :</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" alignment="Qt::AlignLeft">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>Capture rectangle border :</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1" alignment="Qt::AlignRight">
            <widget class="QSpinBox" name="captureRectBorder">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="suffix">
              <string> px</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
